src/layerstack.stanza

```diff
@@ -225,6 +231,11 @@ with:
   constructor => #LayerSpec
   printer => true
 
+defmethod equal? (la:LayerSpec, lb:LayerSpec) :
```

Comment:
Same comment here - Lets not make an explicit `equal?` lets use the `equalable => true` in the defstruct definition.